internal/scsi/generate_mock.sh

```diff
@@ -1,3 +1,15 @@
+# Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
It should be 2020-2022 as we are modifying it now.

Reply:
@abhi16394 have all these comments been addressed?

Reply:
If yes, could you resolve them.

Reply:
changed it

Reply:
Thank you.